### ISSUE TITLE
Is this module ready for production?

### ISSUE THREAD

**Issue by wangyun1517** [google/ngx_brotli#115]
Seems the [module status](https://github.com/google/ngx_brotli#status) is under active development.  
And there is [no official release](https://github.com/google/ngx_brotli/releases) for this.

So is this module ready for production?

**Comment by eustas** [comment]
Will update status soon. Both product seems to be stable. Tagged ngx_brotli recently to make its usage easier.

**Comment by ReillyBrogan** [comment]
Yes, marking this as production-ready and creating source releases (with git submodules resolved) would be very useful to us as well.

**Comment by sreerajkksd** [comment]
Can you also create a tag and release the latest version ? it looks like the latest release (and the only release) present is the release candidate

**Comment by LeviPesin** [comment]
Pretty sure that the main usecase for this module would be to use it in combination with gzip (i.e. fallbacking to gzip when the client does not accept brotli), so I think issues https://github.com/google/ngx_brotli/issues/97, https://github.com/google/ngx_brotli/issues/116, https://github.com/google/ngx_brotli/issues/123 must be fixed before making this module ready for production.

**Comment by LeviPesin** [comment]
Also, it seems like this repository is unmaintained for quite a period of time...

**Comment by beobungbu** [comment]
any updates?

**Comment by eustas** [comment]
Hi. module seems to be quite stable. I need some time to go though PRs / Issues and then I would make a release.
Currently no promises when this will happen. Hopefully soon. Sorry for late responses...

**Comment by LeviPesin** [comment]
@eustas Hi, any updates?

**Comment by Danrancan** [comment]
Any updates on this question?
